### ISSUE TITLE
Reporter [kamon.zipkin.ZipkinReporter] failed to report spans.

### ISSUE THREAD

**Issue by sjoerdmulder** [kamon-io/kamon-zipkin#9]
Updated zipkin to 2.11.10 but it seems that https://github.com/openzipkin/zipkin/commit/a40ebcb47986ed1efb85a0543a17611d78404eb2 caused some extra checks so i get this stacktrace now:
```
java.lang.IllegalArgumentException: parentId is empty
	at zipkin2.Span$Builder.parentId(Span.java:447)
	at kamon.zipkin.ZipkinReporter.convertSpan(Zipkin.scala:58)
	at kamon.zipkin.ZipkinReporter.$anonfun$reportSpans$1(Zipkin.scala:49)
	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:237)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at scala.collection.TraversableLike.map(TraversableLike.scala:237)
	at scala.collection.TraversableLike.map$(TraversableLike.scala:230)
	at scala.collection.AbstractTraversable.map(Traversable.scala:108)
	at kamon.zipkin.ZipkinReporter.reportSpans(Zipkin.scala:49)
	at kamon.ReporterRegistry$Default$SpanReporterTicker.$anonfun$run$9(ReporterRegistry.scala:384)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.util.Try$.apply(Try.scala:213)
	at kamon.ReporterRegistry$Default$SpanReporterTicker.$anonfun$run$8(ReporterRegistry.scala:384)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:658)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run_aroundBody0(Promise.scala:64)
	at scala.concurrent.impl.CallbackRunnable$AjcClosure1.run(Promise.scala:1)
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)
	at kamon.scala.instrumentation.FutureInstrumentation.$anonfun$aroundExecution$1(FutureInstrumentation.scala:46)
	at kamon.Kamon$.withContext(Kamon.scala:120)
	at kamon.scala.instrumentation.FutureInstrumentation.aroundExecution(FutureInstrumentation.scala:46)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:63)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```
What is the cause? Should we sent `null` as parentID? is it because of the `NoIdentifier` Identifier that has an empty string as value?

**Comment by sjoerdmulder** [comment]
@ivantopo any idea / time to fix this? I can help if you want. Would converting empty string to null in the ZipkinReporter be good solution?

**Comment by ivantopo** [comment]
Hey @sjoerdmulder, got your PR merged and released as [v1.0.1](https://github.com/kamon-io/kamon-zipkin/releases/tag/v1.0.1), thanks a lot for the help!
